Hw6_1.py

```diff
@@ -0,0 +1,36 @@
+class Person(object):
+    def __init__(self, name, age, money):
+        self.name = name
+        self.age = age
+        self.money = money
+        print("Information about the person:", name, age, money)
+
+
+class Dealer(Person):
+    def start_game(self):
+        print("The game began")
+
+
+class Gamer(Person):
```

Comment:
he should have not win and loose methods but get money and spend_money ones. 
who has wins and amount of money should be desided by a dealer